src/plot_api/to_image.js

```diff
@@ -138,6 +138,7 @@ function toImage(gd, opts) {
     // extend config for static plot
     var configImage = Lib.extendFlat({}, config, {
         staticPlot: true,
+        exportedPlot: true,
```

Comment:
This here means that #3260 is only fixed for `Plotly.toImage` and `Plotly.downloadImage`, the other (unofficial) methods on `Plotly.Snapshot` will still be buggy.